docs/supported-libraries.md

```diff
@@ -225,6 +225,15 @@ For this reason, the following instrumentations are disabled by default:
 - `jdbc-datasource` which creates spans whenever the `java.sql.DataSource#getConnection` method is called.
 - `dropwizard-metrics` which might create a very low quality metrics data, because of lack of label/attribute support
   in the Dropwizard metrics API.
+- `jaxws-jws-api`
+- `mybatis`
+- `spring-boot-actuator-autoconfigure`
+- `spring-ws`
+- `spring-batch`
+- `spring-security-config-webflux`
+- `spring-security-config-servlet`
```

Comment:
I think these two don't really fit here. Although these instrumentations are disabled by default they are enabled when any of `otel.instrumentation.common.enduser.id.enabled`, `otel.instrumentation.common.enduser.role.enabled` or `otel.instrumentation.common.enduser.scope.enabled` is set.